app/charts/combo/combo-line-column-state.tsx

```diff
@@ -147,8 +149,31 @@ const useComboLineColumnState = (
         TICK_PADDING
     )
   );
+
+  const axisTitleLeft = variables.y.left.label;
+  const axisTitleRight = variables.y.right.label;
+  const axisTitleWidthLeft =
+    getTextWidth(axisTitleLeft, { fontSize: TICK_FONT_SIZE }) + TICK_PADDING;
+  const axisTitleWidthRight =
+    getTextWidth(axisTitleRight, { fontSize: TICK_FONT_SIZE }) + TICK_PADDING;
+  const overLappingTitles = axisTitleWidthLeft + axisTitleWidthRight > width;
+  const overLappingAmount = (axisTitleWidthLeft + axisTitleWidthRight) / width;
+
+  const axisTitleAdjustment =
+    (overLappingTitles
+      ? TICK_FONT_SIZE * Math.ceil(overLappingAmount)
+      : TICK_FONT_SIZE + TITLE_VPADDING) *
+      2 -
+    TICK_FONT_SIZE * 2;
+
+  const topMarginAxisTitleAdjustment = 50 + axisTitleAdjustment;
   const right = Math.max(maxRightTickWidth, 40);
```

Comment:
Nit: I think this could be extracted, as the logic seems to be the same between combo line column and combo line dual charts 👍

Reply:
yup